frontend/tailwind.config.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
     "./src/pages/Authentication/*.tsx",
     "./src/tw-components/*.tsx",
     "./src/pages/LandingPage/*.tsx",
+    "./src/pages/PrivacyPolicyPage/*"
```

Comment:
Looks like the linter is still angry this time about Prettier

My guess is it wants that trailing comma on the last item of the `content` array here like it had before

![image](https://github.com/hackforla/CivicTechJobs/assets/73561520/b52e57bb-74f1-4add-907b-3ba87073a32b)

Reply:
I agree with @MattPereira I ran prettier on this file and it added a trailing comma here.